js/visualizer.js

```diff
@@ -663,50 +662,10 @@ function createCategoryControls(target, criteria) {
       step: 0.05,
     })
     .on("change", function (criterion, index) {
-      // Note: this mutates the critera object
-      criterion["weight"] = this.value;
-      updateSidebar({}, criteria);
-
-      // TODO: Regenerate hospital colors
-    })
-}
-
-function bindControls(criteria) {
```

Comment:
Are there any errors in the log/ can I see your screen? I can't replicate the error on my end.